xml/yast2_userman.xml

```diff
@@ -1297,4 +1297,35 @@
     </varlistentry>
   </variablelist>
  </sect1>
+ <sect1 xml:id="reserved-uids">
+  <title>Reserved user and group IDs</title>
+  <para>When a user creates a file, they automatically become the owner of that file.
+    Additionally, the file is associated with a group, and the creating user's primary group typically becomes the group owner.
+    Files are assigned separate <literal>read</literal>, <literal>write</literal> and <literal>execute</literal> permissions.
+    The root user can change the file owner. Both the root user and the file owner can change access permissions to the file.
+    A regular user can change the group ownership of a file they own only if they are a member of that group.
+  </para>
+  <para>Every user has a UID (unique user ID), which is a numerical identification number. Additionally, each group is associated with a GID (group ID).
+    Users belonging to a group share the same <literal>read</literal>, <literal>write</literal>  and <literal>execute</literal> permissions for any files owned by that group.
+  &productname; reserves the following user and group IDs:
+  </para>
+  <itemizedlist mark="bullet" spacing="normal">
+    <listitem><para><literal>0</literal>: reserved for <literal>root</literal> user</para></listitem>
+    <listitem><para><literal>5</literal>: reserved for <literal>tty</literal> group</para></listitem>
```

Comment:
Why call this out? It doesn't matter in this context, just call it 1 to 999 for system users.